test/test_browser.py

```diff
@@ -584,7 +604,13 @@ def test_preload_caching(self, extra_size):
         fclose(f);
         printf("|%%s|\n", buf);
 
-        assert(strcmp("load me right before", buf) == 0);
+        if (strcmp("load me right before", buf)) {
+          puts("bad data");
+          // Return a code that is easily differentiable from the 0, 1 that
+          // the main test returns, which is the # of cached packages.
+          emscripten_force_exit(99);
+        }
```

Comment:
What is the actual problem here?    I think `assert` is generally better since it send the line number and filename back.   I think it should trigger onAbort which should never be confused with `exit:N`.

Reply:
Ok, reverted.